presto-main/src/main/java/com/facebook/presto/execution/SqlStageExecution.java

```diff
@@ -513,6 +541,11 @@ else if (taskState == TaskState.FINISHED) {
         }
     }
 
+    private static boolean isRecoverableFailure(ExecutionFailureInfo executionFailureInfo)
+    {
+        return true;
```

Comment:
This is for hack/test 😃

Reply:
I added some error codes and we could start from there.